### PR TITLE
fix(translations): Date range filter

### PR DESCRIPTION
**Pull request by fatiiates**
Merges #26074. Extended to test whether pipeline bugs in #26074 persist.

### CODE REVIEW

**Comment by fatiiates** [comment]
Hey @rusackas, there was a development in #26074 and it was stuck because of the CI/CD problems. I merged it into updated superset repository and run the workflow again. Now it works, I thought you could help us about the PR and approval process.

File: superset/translations/de/LC_MESSAGES/messages.json [hunk 1]
```diff
@@ -284,6 +283,9 @@
       "A valid color scheme is required": [
         "Ein gültiges Farbschema ist erforderlich"
       ],
+      "A waterfall chart is a form of data visualization that helps in understanding\n          the cumulative effect of sequentially introduced positive or negative values.\n          These intermediate values can either be time based or category based.": [
```
Comment (john-bodley) [inline superset/translations/de/LC_MESSAGES/messages.json:286]: Why do we need newlines and spaces here?

**Review by john-bodley — COMMENTED** [review]
Thanks @fatiiates for the change. 

Out of interest any reason you didn't just push a commit to the original PR which was failing CI? Additionally it seems like your PR title is somewhat misleading at the change contains more than simply addressing various translations for the date range filter.

**Comment by fatiiates** [comment]
@john-bodley thank you for your help.

I wast just opened this to try the workflow and to solve if there is an error with containerization. LGTM now, we can close this and continue with the #26074

File: superset/translations/de/LC_MESSAGES/messages.json (hunk 1 above)
Comment (fatiiates) [reply superset/translations/de/LC_MESSAGES/messages.json:286]: @john-bodley could you please move this question to #26074? @Ralkion can help you with this.

Comment (Ralkion) [reply superset/translations/de/LC_MESSAGES/messages.json:286]: The short answer is that this is how it's built in the code. The first entry of each array is the 'key' that is used for lookup to get the 'value'.

In this case, you can trace the key via the messages.pot file (superset/translations/messages.pot), which is a generated file. The pot file has a comment indicating where the string originated - which for this message is (superset-frontend/plugins/plugin-chart-echarts/src/Waterfall/index.ts:53)

**Pull request closed without merging**